src/cascadia/TerminalSettingsModel/GlobalAppSettings.h

```diff
@@ -85,6 +85,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         GETSET_SETTING(bool, AlwaysOnTop, false);
         GETSET_SETTING(Model::TabSwitcherMode, TabSwitcherMode, Model::TabSwitcherMode::InOrder);
         GETSET_SETTING(bool, DisableAnimations, false);
+        GETSET_SETTING(bool, DisableProgressRing, false);

```

Comment:
quick check on: why is this a setting? is it too specific? should we have an enum for progressIndicatorStyle? `tab|taskbar|all|none`? I don't know!

Reply:
That's... a fair point. Given that browsers/Visual Studio/other things that have progress indicators do _not_ offer a way to disable them (as far as I know), I'm thinking we remove the setting.

Thoughts?